exporter/common_test.go

```diff
@@ -56,11 +102,23 @@ func TestListCollections(t *testing.T) {
 		"testdb02": {"col01", "col02"},
 	}
 
-	// List all collections in testdb01 (inDBs[0]) but only col01 and col02 from testdb02.
-	filterInNameSpaces := []string{inDBs[0], inDBs[1] + ".col01", inDBs[1] + ".col02"}
-	namespaces, err := listAllCollections(ctx, client, filterInNameSpaces, systemDBs)
-	assert.NoError(t, err)
-	assert.Equal(t, wantNS, namespaces)
+	t.Run("With namespaces list", func(t *testing.T) {
+		// List all collections in testdb01 (inDBs[0]) but only col01 and col02 from testdb02.
+		filterInNameSpaces := []string{inDBs[0], inDBs[1] + ".col01", inDBs[1] + ".col02"}
+		namespaces, err := listAllCollections(ctx, client, filterInNameSpaces, systemDBs)
+		assert.NoError(t, err)
+		assert.Equal(t, wantNS, namespaces)
+	})
+
+	t.Run("Empty namespaces list", func(t *testing.T) {
+		wantNS := map[string][]string{
```

Comment:
Variable shadowing happening here. Can we move this variable to outer scope or move earlier variable to scope of subtest?